packages/react-components/react-components/package.json

```diff
@@ -83,7 +83,8 @@
     "@swc/helpers": "^0.5.1",
     "@fluentui/react-message-bar": "^9.0.22",
     "@fluentui/react-breadcrumb": "^9.0.17",
-    "@fluentui/react-aria": "^9.10.0"
+    "@fluentui/react-aria": "^9.10.0",
+    "@fluentui/react-rating": "9.0.0"
```

Comment:
Ahh, I did use it, but I erroneously changed the versioning because I thought that the alpha versioning was incorrect. I will rerun the script and update this properly